prost::Message::default_instance not found

I am temporarily limited to the 0.6.2 to solve the problem, expect protobuf-build to be stable as soon as possible

[Related Job](https://travis-ci.org/pingcap/grpc-rs/jobs/552317358)

Got the same error when regening raft-rs/raft-proto:

```
~/git-repos/raft-rs/proto λ> cargo build --features gen
   Compiling raft-proto v0.1.0 (/home/ice/git-repos/raft-rs/proto)
error[E0599]: no function or associated item named `default_instance` found for type `dyn prost::eraftpb::Entry_MESSAGE::_prost::Message` in the current scope
 --> proto/src/prost/wrapper_eraftpb.rs:6:27
  |
6 |         ::prost::Message::default_instance()
  |                           ^^^^^^^^^^^^^^^^ function or associated item not found in `dyn prost::eraftpb::Entry_MESSAGE::_prost::Message`

error[E0599]: no function or associated item named `default_instance` found for type `dyn prost::eraftpb::Entry_MESSAGE::_prost::Message` in the current scope
   --> proto/src/prost/wrapper_eraftpb.rs:126:27
    |
126 |         ::prost::Message::default_instance()
    |                           ^^^^^^^^^^^^^^^^ function or associated item not found in `dyn prost::eraftpb::Entry_MESSAGE::_prost::Message`

error[E0599]: no function or associated item named `default_instance` found for type `dyn prost::eraftpb::Entry_MESSAGE::_prost::Message` in the current scope
   --> proto/src/prost/wrapper_eraftpb.rs:272:27
    |
272 |         ::prost::Message::default_instance()
    |                           ^^^^^^^^^^^^^^^^ function or associated item not found in `dyn prost::eraftpb::Entry_MESSAGE::_prost::Message`

error[E0599]: no function or associated item named `default_instance` found for type `dyn prost::eraftpb::Entry_MESSAGE::_prost::Message` in the current scope
   --> proto/src/prost/wrapper_eraftpb.rs:388:27
    |
388 |         ::prost::Message::default_instance()
    |                           ^^^^^^^^^^^^^^^^ function or associated item not found in `dyn prost::eraftpb::Entry_MESSAGE::_prost::Message`

error[E0599]: no function or associated item named `default_instance` found for type `dyn prost::eraftpb::Entry_MESSAGE::_prost::Message` in the current scope
   --> proto/src/prost/wrapper_eraftpb.rs:558:27
    |
558 |         ::prost::Message::default_instance()
    |                           ^^^^^^^^^^^^^^^^ function or associated item not found in `dyn prost::eraftpb::Entry_MESSAGE::_prost::Message`

error[E0599]: no function or associated item named `default_instance` found for type `dyn prost::eraftpb::Entry_MESSAGE::_prost::Message` in the current scope
   --> proto/src/prost/wrapper_eraftpb.rs:642:27
    |
642 |         ::prost::Message::default_instance()
    |                           ^^^^^^^^^^^^^^^^ function or associated item not found in `dyn prost::eraftpb::Entry_MESSAGE::_prost::Message`

error[E0599]: no function or associated item named `default_instance` found for type `dyn prost::eraftpb::Entry_MESSAGE::_prost::Message` in the current scope
   --> proto/src/prost/wrapper_eraftpb.rs:738:27
    |
738 |         ::prost::Message::default_instance()
    |                           ^^^^^^^^^^^^^^^^ function or associated item not found in `dyn prost::eraftpb::Entry_MESSAGE::_prost::Message`

error: aborting due to 7 previous errors

For more information about this error, try `rustc --explain E0599`.
error: Could not compile `raft-proto`.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "protobuf-build"
-version = "0.6.3"
+version = "0.6.4"
 authors = ["Nick Cameron <nrc@ncameron.org>"]
 edition = "2018"
 license = "Apache-2.0"

--- a/src/wrapper.rs
+++ b/src/wrapper.rs
@@ -139,7 +139,7 @@ where
         .collect::<Result<Vec<_>, _>>()?;
     writeln!(buf, "}}")?;
     if gen_opt.contains(GenOpt::MESSAGE) {
-        generate_message_trait(&item.ident, prefix, buf, gen_opt.contains(GenOpt::NEW))?;
+        generate_message_trait(&item.ident, prefix, buf)?;
     }
     Ok(())
 }
@@ -185,18 +185,7 @@ where
     if gen_opt.contains(GenOpt::MESSAGE) {
         writeln!(
             buf,
-            "#[inline] pub fn default_ref() -> &'static Self {{ ::prost::Message::default_instance() }}",
-        )
-    } else if gen_opt.contains(GenOpt::NEW) {
-        writeln!(
-            buf,
-            "#[inline] pub fn default_ref() -> &'static Self {{
-                ::lazy_static::lazy_static! {{
-                    static ref INSTANCE: {0}{1} = {0}{1}::new_();
-                }}
-                &*INSTANCE
-            }}",
-            prefix, name,
+            "#[inline] pub fn default_ref() -> &'static Self {{ ::protobuf::Message::default_instance() }}",
         )
     } else {
         writeln!(
@@ -212,12 +201,7 @@ where
     }
 }
 
-fn generate_message_trait<W>(
-    name: &Ident,
-    prefix: &str,
-    buf: &mut W,
-    has_new: bool,
-) -> Result<(), io::Error>
+fn generate_message_trait<W>(name: &Ident, prefix: &str, buf: &mut W) -> Result<(), io::Error>
 where
     W: Write,
 {
@@ -245,31 +229,17 @@ where
         buf,
         "fn descriptor(&self) -> &'static ::protobuf::reflect::MessageDescriptor {{ Self::descriptor_static() }}",
     )?;
-    if has_new {
-        writeln!(buf, "fn new() -> Self {{ Self::new_() }}",)?;
-        writeln!(
-            buf,
-            "fn default_instance() -> &'static {}{} {{
-            ::lazy_static::lazy_static! {{
-                static ref INSTANCE: {0}{1} = {0}{1}::new_();
-            }}
-            &*INSTANCE
-        }}",
-            prefix, name,
-        )?;
-    } else {
-        writeln!(buf, "fn new() -> Self {{ Self::default() }}",)?;
-        writeln!(
-            buf,
-            "fn default_instance() -> &'static {}{} {{
-            ::lazy_static::lazy_static! {{
-                static ref INSTANCE: {0}{1} = {0}{1}::default();
-            }}
-            &*INSTANCE
-        }}",
-            prefix, name,
-        )?;
-    }
+    writeln!(buf, "fn new() -> Self {{ Self::default() }}",)?;
+    writeln!(
+        buf,
+        "fn default_instance() -> &'static {}{} {{
+        ::lazy_static::lazy_static! {{
+            static ref INSTANCE: {0}{1} = {0}{1}::default();
+        }}
+        &*INSTANCE
+    }}",
+        prefix, name,
+    )?;
     // The only way for this to be false is if there are `required` fields, but
     // afaict, we never use that feature. In any case rust-protobuf plans to
     // always return `true` in 3.0.

